### PR TITLE
[ADD] Automatic nginx + SSL installation

### PR DESCRIPTION
**Pull request by tarteo**

### CODE REVIEW

File: odoo_install.sh [hunk 1]
```diff
@@ -253,3 +267,24 @@ echo "Start Odoo service: sudo service $OE_CONFIG start"
...
+    sudo ln -s /etc/nginx/sites-available/odoo /etc/nginx/sites-enabled/odoo
+
+    echo -e "\n---- SSL certificate ----"
+    sudo apt install software-properties-common -y
+    sudo add-apt-repository universe -y
+    sudo add-apt-repository ppa:certbot/certbot -y
+    sudo apt update
+
+    sudo apt install certbot
+    sudo certbot certonly --standalone --preferred-challenges http -d $DOMAIN -n --agree-tos --email info@onestein.nl
```
Comment (astirpe) [inline odoo_install.sh]: Can you make email info@onestein.nl configurable? For example in a variable?

Moreover: did you try if this line works without first restarting nginx service (seems you didn't restart it before)?

File: odoo_install.sh [hunk 2]
```diff
@@ -253,3 +267,24 @@ echo "Start Odoo service: sudo service $OE_CONFIG start"
...
+    
+    sudo service nginx restart
```
Comment (astirpe) [inline odoo_install.sh]: If nginx is already started, a `sudo service nginx reload` would be enough here.

**Review by astirpe — APPROVED** [review]
I made a quick review without testing it. LGTM!
I will test it as soon as I will have do a complete installation of a server.

My personal preference would be to put the `nginx` file directly inside `odoo_install.sh`, instead of keeping it separate. This will avoid problems if someone (like me) forgets to `wget` it.

Feel free to merge when you want.

**Comment by tarteo** [comment]
@astirpe Thanks for the review!
> If nginx is already started, a sudo service nginx reload would be enough here.

nginx needs to be off I think, because --standalone starts a built-in webserver.

>   Can you make email info@onestein.nl configurable? For example in a variable?

:+1:

> My personal preference would be to put the nginx file directly inside odoo_install.sh, instead of keeping it separate. This will avoid problems if someone (like me) forgets to wget it.

I've put the wget in the script itself. This way we can't forget it :smile: 

Probably needs some fine tuning when we actually use it, but it's OK for now. It doesn't obstruct the normal Odoo script so I'll merge.

**Pull request merged**